concrete/src/Block/Block.php

```diff
@@ -1648,6 +1648,11 @@ public function duplicate($nc, $isCopyFromMasterCollectionPropagation = false)
                 );
         }
 
+        $event = $app->make(\Concrete\Core\Block\Events\BlockDuplicate::class, ['subject' => $nb]);
```

Comment:
Can we simply make this `new BlockEvent($nb)` ? There's no reason to use dependency injection here.

Reply:
@aembler changed in 454f09afc2465f8aeae446aa023f477f0163d665